.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+Thank you for reporting an issue with **AlexaPi**.
+
+Make sure you are running the latest version of **AlexaPi**, and look through [issues](https://github.com/alexa-pi/AlexaPi/issues?utf8=%E2%9C%93&q=) before submitting.
+
+Check out our [wiki](https://github.com/alexa-pi/AlexaPi/wiki), and [Audio Debugging](https://github.com/alexa-pi/AlexaPi/wiki/Audio-setup-&-debugging) in particular before filing an issue.
+
+In order to help troubleshooting, be sure to include the following information:
```

Comment:
Put an item something like:
Output of the [audio debugging script](link to the section on the wiki).
Same style as other, just nothing to fill and no colon, just ".".